inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/impl/InlongClientImpl.java

```diff
@@ -163,12 +166,19 @@ public Map<String, InlongGroupStatusInfo> listGroupStatus(List<String> groupIds,
         List<SortStatusInfo> sortStatusInfos = groupClient.listSortStatus(statusRequest);
 
         if (CollectionUtils.isNotEmpty(sortStatusInfos)) {
-            Map<String, SortStatus> sortStatusMap = sortStatusInfos.stream()
-                    .collect(Collectors.toMap(SortStatusInfo::getInlongGroupId, SortStatusInfo::getSortStatus));
-            groupStatusMap.forEach((groupId, groupStatusInfo) -> groupStatusInfo
-                    .setSortStatus(sortStatusMap.getOrDefault(groupId, SortStatus.NOT_EXISTS)));
+            Map<String, SortStatusInfo> sortStatusInfoMap = sortStatusInfos.stream()
+                    .collect(Collectors.toMap(SortStatusInfo::getInlongGroupId, Function.identity()));
+            groupStatusMap.forEach((groupId, groupStatusInfo) -> {
+                groupStatusInfo.setSortStatus(SortStatus.NOT_EXISTS);
```

Comment:
Why not use `sortStatusMap.getOrDefault(groupId, SortStatus.NOT_EXISTS)`?

Reply:
> Why not use `sortStatusMap.getOrDefault(groupId, SortStatus.NOT_EXISTS)`?

Because other properties of SortStatusInfo are used，the default is also the status SortStatus.NOT_EXISTS